lib/chef/resource/windows_task.rb

```diff
@@ -59,6 +59,12 @@ class WindowsTask < Chef::Resource
       property :execution_time_limit, [String, Integer], default: "PT72H" # 72 hours in ISO8601 duration format
       property :minutes_duration, [String, Integer]
       property :minutes_interval, [String, Integer]
+      property :disallow_start_on_battery, [TrueClass, FalseClass], default: false,
+               introduced: "14.4",
+               description: "Scheduled task option when system is on AC power."
+      property :stop_on_battery, [TrueClass, FalseClass], default: false,
```

Comment:
Can we change this description to "Stop the task when the system switches to battery power"